packages/design-picker/src/components/index.tsx

```diff
@@ -91,13 +91,15 @@ const DesignButton: React.FC< DesignButtonProps > = ( {
 	const blankCanvasTitle = __( 'Blank Canvas', __i18n_text_domain__ );
 	const designTitle = isBlankCanvas ? blankCanvasTitle : defaultTitle;
 
-	const badgeType = design.is_premium ? 'premium' : 'none';
+	const isPremiumDesign = design?.design_tier === 'premium';
```

Comment:
I don't thing we have any premium theme on the design picker, maybe it's just there in case they get added or something 🤷‍♂️

Reply:
So this premium means premium as in a theme that requires the ex-Premium plan? Or is it supposed to mean a Premium theme that requires a purchase of the actual theme?

Reply:
It's premium as in non-free, but this code is only used on the DIFM design picker and only where `hideDescription` / `hideBadge` are false. They're [currently hard-coded to true](https://github.com/Automattic/wp-calypso/blob/4a5dbf775ac04f34984788cd70a2718f531378fe/client/signup/config/steps-pure.js#L864-L865), presumably because using DIFM requires the premium plan or higher anyway.

Enabling those constants makes the DIFM design picker look broken:

<img width="1452" alt="Screenshot 2024-07-04 at 15 44 23" src="https://github.com/Automattic/wp-calypso/assets/93301/af408580-fe4d-4af4-ac23-e7f8f85d865e">